### PR TITLE
PieceJustificative: fix gestionnaire view when pj champ is not filled

### PR DESCRIPTION
**Pull request by LeSim**

### CODE REVIEW

File: app/views/new_gestionnaire/dossiers/_champs.html.haml [hunk 1]
```diff
@@ -35,8 +35,9 @@
             = "#{c.libelle} :"
           %td.rich-text
             - pj = c.piece_justificative_file
-            %a{ href: url_for(pj), target: '_blank' }
-              = pj.filename.to_s
+            - if pj.attached?
```
Comment (gregoirenovel) [inline app/views/new_gestionnaire/dossiers/_champs.html.haml]: Rajouter un `else` ?

Comment (LeSim) [reply app/views/new_gestionnaire/dossiers/_champs.html.haml]: genre `non fourni` ?

Comment (gregoirenovel) [reply app/views/new_gestionnaire/dossiers/_champs.html.haml]: "Pièce justificative non fournie"

**Comment by LeSim** [comment]
up

**Pull request merged**